sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -115,12 +117,18 @@ def __init__(self, name, init_config, agentConfig, instances=None):
             'rate' : self.rate,
             'histogram': self.histogram
         }
+        self.adoprovider = self.default_adoprovider
 
         self.connector = init_config.get('connector', 'adodbapi')
         if not self.connector.lower() in self.valid_connectors:
             self.log.error("Invalid database connector %s, defaulting to adodbapi" % self.connector)
             self.connector = 'adodbapi'
 
+        self.adoprovider = init_config.get('adoprovider', self.default_adoprovider)
+        if not self.adoprovider.upper() in self.valid_adoproviders:
```

Comment:
Just a small nit, can we rearrange this so it's `if self.adoprovider.upper() not in self.valid_adoproviders:`